### ISSUE TITLE
Generation does not work with splitted files

### ISSUE THREAD

**Issue by jbellmann** [zalando-stups/swagger-codegen-tooling#23]

**Comment by XtreZz** [comment]
any news on this ? i also have this problem

**Comment by jbellmann** [comment]
Not yet. Had no time to look into it the last 3 days. Sorry.

But could you provide a simple 2 file example swagger-definition please. That could improve speed when resolving this issue ;-)

**Comment by stephenlynch** [comment]
Hi,

I am also having this issue. The generation is working but not for collections with generics:

```json
"fooList": {
        "type": "object",
        "additionalProperties": {
          "type": "string"
        }
      },
```
Should look like this:
```java
private Map<String, String> fooList;
```

but generates this:
```java
private String fooList;
```

**Comment by jbadeau** [comment]
Any idea if this will be fixed.

**Comment by jbellmann** [comment]
Hi, I had a look into the code-generator documentation and did not found any hint that it is possible to use more than one spec-file (it's the -i parameter, right?). So we only could build a custom file-merge and present that to the code-generator.

Do you have more information about using multiple spec-files for the code-generator?

**Comment by jbadeau** [comment]
In swagger specs you have the option to reference external elements like:

`$ref: '../common/Definitions.yaml#/Error'`

**Comment by jbellmann** [comment]
Yes I read that also somewhere in the net, and also that the 'swagger-editor' supports that. But the 'code-generator' only supports one, right? And we only delegate to that generator. I will try it.

**Comment by jbadeau** [comment]
The swagger generator supports it. Yes, I think you need only delegate.

**Comment by gitLinda** [comment]
I'm also interested on this fix
